setup.py

```diff
@@ -9,13 +9,10 @@ def read(fname1, fname2):
         fname = fname2
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
-extra = {}
-if sys.version_info >= (3,):
-    extra['use_2to3'] = True
 
 setup(
     name = "django-async",
-    version = "0.6.0.18",
+    version = "0.7.0.0",
```

Comment:
Can you set this to 0.6.0.19? I'll change to 0.6.1 when I pull to develop and then to 0.7 when I push it to master and release it. If nothing comes up then I'll call this 1.0